### PR TITLE
Trying to only bring primitives over

### PR DESCRIPTION
**Pull request by maryamariyan**
cc: @adamsitnik

### CODE REVIEW

File: src/benchmarks/micro/libraries/Microsoft.Extensions.Primitives/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Licensed to the .NET Foundation under one or more agreements.
```
Comment (adamsitnik) [inline src/benchmarks/micro/libraries/Microsoft.Extensions.Primitives/Properties/AssemblyInfo.cs:1]: this file can be removed

File: src/benchmarks/micro/MicroBenchmarks.csproj [hunk 2]
```diff
@@ -23,21 +24,59 @@
...
+      <Link>Shared\ParamsDisplayInfoColumn.cs</Link>
+    </Compile>
+    <Compile Include="$(BenchmarkRunnerPath)BenchmarkRunner\Program.cs">
+      <Link>Shared\Program.cs</Link>
+    </Compile>
+    <Compile Include="C:\CodeHub\performance\src\benchmarks\micro\libraries\Internal\ServiceProviderMode.cs">
+      <Link>Internal\ServiceProviderMode.cs</Link>
+    </Compile> -->
+  </ItemGroup>
+
```
Comment (adamsitnik) [inline src/benchmarks/micro/MicroBenchmarks.csproj:79]: the referenced files were related to custom configs created by ASP.NET, here we are using our own config: https://github.com/dotnet/performance/blob/master/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs

so I think that you can just remove it

```suggestion
```

File: src/benchmarks/micro/MicroBenchmarks.csproj [hunk 3]
```diff
@@ -12,8 +12,9 @@
     <DebugType>portable</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
+    <!-- <BenchmarkRunnerPath>C:\CodeHub\performance\src\benchmarks\micro\libraries\BenchmarkRunner\</BenchmarkRunnerPath> -->
```
Comment (adamsitnik) [inline src/benchmarks/micro/MicroBenchmarks.csproj:15]: the `BenchmarkRunnerPath` should not be needed here

```suggestion
```

**Comment by maryamariyan** [comment]
Closing this in favor of https://github.com/dotnet/performance/pull/1288

**Pull request closed without merging**